oneview_redfish_toolkit/api/zone.py

```diff
@@ -87,6 +88,15 @@ def __init__(self,
 
         self._validate()
 
+    def get_zone_name(self, spt_name, resource_id):
+        zone_name = spt_name
+        resource_id_group = resource_id.split("-")
+        # if resource group len is more than 5
+        # it means that exists an enclosure
+        if len(resource_id_group) > 5:
```

Comment:
In both cases (using enclosure name or enclosure id) we can avoid doing this code by receiving the enclosure name (or enclosure id) as a parameter of constructor, because we already have that [information](https://github.com/HewlettPackard/oneview-redfish-toolkit/blob/master/oneview_redfish_toolkit/blueprints/zone.py#L41) in zone blueprint.